docs/resources/extended-query-tags.md

```diff
@@ -0,0 +1,214 @@
+# Extended Query Tags
+
+Extended query tags allows querying over DICOM tags that are not supported by the DICOM Web standard for [QIDO-RS](). By enabling this feature, it is possible to query against both publicly defined or "standard" DICOM tags as well as private ones.
```

Comment:
Potentially clarify that you can query against tags supported by QIDO, standard tags not supported by QIDO and private tags.